src/libostree/ostree-repo.c

```diff
@@ -3033,6 +3035,10 @@ reload_core_config (OstreeRepo          *self,
       }
   }
 
+  if (!ot_keyfile_get_boolean_with_default (self->config, "fsverity", "required",
```

Comment:
Maybe `ex-fsverify` to mark it experimental for now?

Reply:
Yeah, makes sense.